definitions/3.0.0/anySchema.json

```diff
@@ -1,16 +1,35 @@
 {
-  "if": {
-    "required": [
-      "schema"
-    ]
-  },
-  "then": {
-    "$ref": "http://asyncapi.com/definitions/3.0.0/multiFormatSchema.json"
-  },
-  "else": {
-    "$ref": "http://asyncapi.com/definitions/3.0.0/schema.json"
-  },
-  "description": "An object representing either a schema or a multiFormatSchema based on the existence of the 'schema' property. If the property 'schema' is present, use the multi-format schema. Use the default AsyncAPI Schema otherwise.",
+  "$id": "http://asyncapi.com/definitions/3.0.0/anySchema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "http://asyncapi.com/definitions/3.0.0/anySchema.json"
+  "type": "object",
+  "description": "An object representing either a Reference, a Schema or a Multi Format Schema",
+  "properties": {
+    "schemaFormat": {
+      "type": "string",
+      "description": "Supported Schema format"
+    },
+    "schema": {
+      "type": "object",
+      "description": "Schema definition"
+    },
+    "$ref": {
+      "type": "string",
+      "description": "Reference to schema"
+    }
+  },
```

Comment:
This change should no longer be needed, original file should work fine :v:

Reply:
I removed properties, thanks for suggestion)